.github/scripts/filter_test_configs.py

```diff
@@ -177,18 +159,21 @@ def filter(test_matrix: Dict[str, List[Any]], labels: Set[str]) -> Dict[str, Lis
 
         label = f"{PREFIX}{config_name.strip()}"
         if label in labels:
-            print(
-                f"Select {config_name} because label {label} is presented in the pull request by the time the test starts"
-            )
+            msg = f"Select {config_name} because label {label} is presented in the pull request by the time the test starts"
```

Comment:
I know this is carried over from the previous, but I think it should be `present in the pull request`